apiary.apib

```diff
@@ -3301,6 +3303,7 @@ Get a list of tasks.
             + completed: true (boolean, optional)
             + due_by: `2019-02-04` (string, optional)
             + due_from: `2019-10-20` (string, optional)
+            + term: `website design` (string, optional) - Searches for a term in the description.
```

Comment:
If it only searches in the description, shouldn't we name the filter `description`?

Reply:
It currently searches on description but the idea is that it can be extended in the future. Should I remove the `Searches for a term in the description` bit?

Reply:
Ideally, we would have both a `description` and `term` filter so that the end-user can choose how he wants to filter

Reply:
Will make sure to consider that once we extend the search to other fields.